### ISSUE TITLE
Round Robin effect when using with Mozilla IoT

### ISSUE THREAD

**Issue by lacojim** [bwssytems/ha-bridge#1209]
When pulling a device into HAB from Mozilla IoT, the device then shows up back in Mozilla IoT when adding a new device there. Is there a way to potentially avoid this from happening? It can be quite confusing with many devices. Maybe there should an option to avoid Mozilla IoT devices in HABridge from being returned back to IoT?

**Comment by bwssytems** [comment]
Very confused on your question. Could you elaborate on the impact from ha-bridge?

**Comment by lacojim** [comment]
Sorry for the confusion. It is kind of hard to explain. I have Mozilla IoT running on a RPi4 to control my Zigbee devices. HABridge runs full time on a Fedora server.

- I export my x10 devices from HABridge into Mozilla IoT using its Philips Hue add-on.
- I import my Mozilla IoT Zigbee devices into HABridge.

Under "Bridge Control" in HABridge, I added the IP, port, username and pw of the Mozilla IoT connection. I then click on "Mozilla IoT Devices" from HABridge and it shows me all the devices on the RPi4. I then proceed to pull in these Zigbee devices to HAB. They work great! Once added, I go back to Mozilla IoT's web interface on the RPi4 and scan for new devices. The devices I just imported into HAB now show up in M-IoT, even though they were imported from this device.

I was asking if there was a way to keep these devices from looping back into M-IoT, since that was where they originated.  

I hope that is more clear now.
